service/src/main/java/bio/terra/tanagra/db/StudyDao.java

```diff
@@ -58,15 +61,16 @@ public StudyDao(NamedParameterJdbcTemplate jdbcTemplate) {
   @WriteTransaction
   public void createStudy(Study study) {
     final String sql =
-        "INSERT INTO study (study_id, display_name, description, properties) "
-            + "VALUES (:study_id, :display_name, :description, CAST(:properties AS jsonb))";
+        "INSERT INTO study (study_id, display_name, description, properties, created_by) "
+            + "VALUES (:study_id, :display_name, :description, CAST(:properties AS jsonb), :created_by)";
 
     MapSqlParameterSource params =
         new MapSqlParameterSource()
             .addValue("study_id", study.getStudyId())
             .addValue("display_name", study.getDisplayName())
             .addValue("description", study.getDescription())
-            .addValue("properties", DbSerDes.propertiesToJson(study.getProperties()));
+            .addValue("properties", DbSerDes.propertiesToJson(study.getProperties()))
+            .addValue("created_by", study.getCreatedBy());
```

Comment:
You're not setting the `created` field here because its set to default to `now()` in the liquibase file right? If so, a comment here explaining that would be helpful. ditto for the other dao classes

Reply:
Done. I agree comment is better, thanks.